<header class="banner"><div class="banner">
  <div class="logos">
    <div><img title="Julia logo" src="../figures/julia-logo-color.svg" alt="Julia animated logo" width=35 /></div>
    <div>◌</div>
    <div><img title="Powered by MathJax" src="https://www.mathjax.org/badge/mj-logo.svg"
              alt="Powered by MathJax" width=65 />
    </div>
  </div>  <!-- logos -->
  <div class="cite_textbook cb7">4.126 <span class="authors"></span></div>
</div></header>



<div class="mcs-bs-callout mcs-bs-callout-info">

#### work time: 16 minutes

</div></div>

### Given
A mass of 5 kg of saturated liquid–vapor mixture of water is contained in a piston–cylinder device at 125 kPa. Initially, 2 kg of the water is in the liquid phase and the rest is in the vapor phase. Heat is now transferred to the water, and the piston, which is resting on a set of stops, starts moving when the pressure inside reaches 300 kPa. Heat transfer continues until the total volume increases by 20 percent. 

   <img src="../figures/cb7,4-126.png" width=20% />




### Find

1. Show the process on a P-v diagram.
1. Determine the following
  1. the initial and final temperatures,  
  1. the mass of liquid water when the piston first starts moving,  
  1. the work done during this process, and  


### Get Organized   

First, we break the problem down into states and the processes that connect them. We are given an initial state. Heat is added at constant volume until we reach the second state, which is right before the piston lifts. More heat is added at constant pressure until we reach the final state.

We also note that the only work interactions are boundary work.

![Cengelboles7ed4-126](../figures/Cengelboles7ed4-126.png)

Plotting the processes is also very helpful, and a habit you want to pick up.

![Cengelboles7ed4-126](../figures/Cengelboles7ed4-126plot.png)



Let's go through all the information we need to find in order to answer the questions.  

1. The initial and final temperatures  
   1. $T_1 = T_{sat@125\,\rm kPa}$   
   1. We get $T_3$ once we resolve State ⓷ <br><br>

1. The mass of liquid water when the piston first starts moving
   1. This is state ⓶, and  
   1. $\displaystyle m_{liquid,2} = m_{total} - m_{vapor,2} = m_{total} - x_2 m_{total} = \left(1 - x_2 \right)m_{total}$ <br><br>

3. The work done during this process: $W_{total} = W_{1 \rightarrow 2} + W_{2 \rightarrow 3}$   

  States ⓵ and ⓶ are connected by a constant volume process. With no volume change, there is no boundary work, so  $W_{1 \rightarrow 2} = 0$.

$\quad$ Thus, $$ W_{total} = W_{2 \rightarrow 3} = \int^3_2 P_2\, \text dV = P_2 \left( V_3 - V_2 \right) = P_2m \left( v_3 - v_2 \right) = P_2m \left( 1.2 v_1 - v_1 \right) = 0.2 P_2 m v_1 $$  


4. Show the process on a P-v diagram.

  See plot way above



# Attack 

1. The initial and final temperatures
  1. $\color{red} T_1 = T_{sat@125\,\rm kPa} =   105.52\, ℃ $ 
  1. We'll come back to $T_3.$
 
 

2. The mass of liquid water when the piston first starts moving     

  $m_{liquid,2} = \left(1 - x_2 \right)m_{total}$, so we need $x_2: \ce{H2O} \left\{ \begin{align*} \ce{P_2 \ &= \ 300 kPa} \\
    \ce{v_2 \ &= \ v_1 }
    \end{align*} 
    \right.$

  At State ⓵, $\ce{H2O} \left\{ \begin{align*} \ce{P_1 \ &= \ 125 kPa} \\
    x_1 \ &= \ 0.6 
    \end{align*} 
    \right.$   

  Our saturated pressure table for water does not have a breakpoint for 125 kPa, but it does have 1 bar and 1.5 bar. We can interpolate between the two easily, because our desired point is halfway between the breakpoints.

In [1]:
using Unitful

vf_100kPa = 0.0010432u"m^3/kg"
vg_100kPa =     1.694u"m^3/kg"

vf_150kPa = 0.0010528u"m^3/kg"
vg_150kPa =     1.159u"m^3/kg"

vf1 = (vf_100kPa + vf_150kPa)/2  
vg1 = (vg_100kPa + vg_150kPa)/2  

x1 = 0.6

v1 = vf1 + x1*(vg1 - vf1); @show v1;

v1 = 0.8563192 m^3 kg^-1


2. (_cont'd_) The mass of liquid water when the piston first starts moving     

  Back to finding $x_2: \ce{H2O} \left\{ \begin{align*} \ce{P_2 \ &= \ 300 kPa} \\
    \ce{v_2 \ &= \ v_1} = 0.856\ \ce{m^3/kg}
    \end{align*} 
    \right.$  

  We won't presume we know the state's location, so we head to the saturation tables as usual, and find $\ce{v_{g@300kPa} = 0.6058 m^3/kg}$. We know $\ce{v_2} = 0.856\, \ce{m^3/kg}$. Therefore, $\ce{v_2} > \ce{v_{g@300kPa}},$ which means we have superheated vapor, and $ \color{red} m_{liquid,2} = 0.$

3. The work done during this process  

  $W_{total} = 0.2 P_2 m v_1$   

In [ ]:
m = 5u"kg"
P2 = 300u"kPa"

W = 0.2 * P2 * m * v1 |> u"kJ"
@show W;

$\color{red} W_{\rm total} = \pu{256.90kJ}$   

#### back to (a) and $T_3$.  

Remember, $v_3 = 1.2v_1$.

In [ ]:
v3 = 1.2 * v1; @show v3;

$\quad$ Finding $T_3: \ce{H2O} \left\{ \begin{align*} \ce{P_3 \ &= \ 300 kPa} \\
    v_3 \ &= 1.028\ \rm m^3/kg
    \end{align*} 
    \right.$  

$\quad$ We find state ③ on the superheated steam table with a little interpolation. 

| T [℃]   |  v [$\bf m^3/$kg]  |
| --------|:------------------:|
|  360    |        0.969       |  
|  400    |        1.032       | 


In [ ]:
# interpolate

T_360°C = 360u"°C"  # silly, but consistent
v_360°C = 0.969u"m^3/kg"

T_400°C = 400u"°C"
v_400°C = 1.032u"m^3/kg"

T3 = T_360°C + (T_400°C - T_360°C)*(v3 - v_360°C)/(v_400°C - v_360°C) |> u"°C" 
@show T3;

$\color{red} T_3 = 397.20\, \rm °C$ 

<div class="mcs-bs-callout mcs-bs-callout-success extra">

#### Finis. Now summarize.

1. The initial and final temperatures
  
  \begin{align*}
  T_{\rm initial} &= \pu{105.52°C} \\
  T_{\rm final} &= 397.20\, \rm °C
  \end{align*} <br><br>
  
1. The mass of liquid water when the piston first starts moving 
  
  $$m = 0$$ <br><br>
  
1. The work done during this process  
  
  $$W_{\rm total} = \pu{256.90kJ}$$    



  
</div></div>

<footer style="background-color: #461d7c; color:rgba(253,208,35,0.35); line-height:2em; padding-left:1em; padding-right:0.5em; bottom:0; float:bottom; text-align:right;"><i>ψᵟ</i></footer>